src/index.js

```diff
@@ -0,0 +1,147 @@
+// const axios = require('axios');
+const state = {
+    city: 'Seattle',  
+    temperature: 20     
+};
+
+const getCurrentTemperature = () => {
+    const cityName = cityNameInput.value || 'Seattle';  
+
+    // Fetch latitude and longitude for the city
+    axios
+        .get('http://localhost:5000/location', {
+            params: { q: cityName }
+        })
+        .then(response => {
+            const { lat, lon } = response.data[0];  
+            console.log('Latitude:', lat);
+            console.log('Longitude:', lon);
+
+            return axios.get('http://localhost:5000/weather', {
+                params: { lat: lat, lon: lon }
+            });
+        })
+        .then(weatherResponse => {
+            const temperatureInKelvin = weatherResponse.data.main.temp;
+            const temperatureInFahrenheit = parseInt((temperatureInKelvin - 273.15) * 9 / 5 + 32);
+            console.log(`Temperature in Fahrenheit: ${temperatureInFahrenheit}°F`);
+
+            state.temperature = parseFloat(temperatureInFahrenheit, 10);  
+
+            tempValue.textContent = `${state.temperature}°F`;
+
+            headerCityName.textContent = cityName;
+
+            temperatureColor();
+            weatherGardenDisplay();
+        })
+        .catch(error => {
+            console.error('Error fetching weather data:', error);
+        });
+};
+
+const tempValue = document.getElementById('tempValue')
+const gardenSection = document.querySelector('.garden__section ');
+const landscape = document.getElementById('landscape')
+const cityNameInput = document.getElementById('cityNameInput');
+const headerCityName = document.getElementById('headerCityName');
+const currentTemperature = document.getElementById('currentTempButton')
+const skySection = document.querySelector('.sky__section ');
+const skySelect = document.getElementById('skySelect')
+const sky = document.getElementById('sky')
+const cityNameReset = document.getElementById('cityNameReset')
+
```

Comment:
Looks like we are doing similar logic in a few places (finding an element by `id` and then, later in our code, adding an event handler to it for a specific event). Do you think that this could call for a helper function to make our code a bit more DRY?